library/kapacitor

```diff
@@ -10,9 +10,17 @@ Directory: kapacitor/1.4
 Tags: 1.4-alpine, 1.4.1-alpine
 Directory: kapacitor/1.4/alpine
 
-Tags: 1.5, 1.5.9, latest
+Tags: 1.5, 1.5.9
 Architectures: amd64, arm32v7, arm64v8
 Directory: kapacitor/1.5
 
-Tags: 1.5-alpine, 1.5.9-alpine, alpine
+Tags: 1.5-alpine, 1.5.9-alpine
 Directory: kapacitor/1.5/alpine
+
+Tags: 1.6, 1.6.1, latest
+Architectures: amd64, arm32v7, arm64v8
```

Comment:
This should probably be minus `arm32v7`, right?  (looks like `armhf` was dropped from the relevant `case` statement in this version)

Reply:
thanks.